pr_parser.py

```diff
@@ -217,100 +158,119 @@ def get_all_related_prs(self, repo, sha, pull_id, actual_commit):
             #avoid endless loop
             if _tmp_pr:
                 _tmp_pr = [t for t in _tmp_pr if t not in all_prs]
-            all_prs.extend(_tmp_pr)    
+            all_prs.extend(_tmp_pr)
             base_pr = _tmp_pr
 
         return all_prs
+
+    def get_under_test_prs(self):
+        """
+        According to the self.__pr_connectivity_map and self.__pr_list
+        to find those under_test_prs.
+        under_test_pr: 
+            1. root pr(which trigger this build) is under_test
+            2. pr that diconnected with under_test_pr is under_test_pr
+        """
+        # the final filtered results
+        # under_test_prs to be parsed to find related under_test_prs
+        tmp_under_test_prs = [self.__pr_list[0]]
+        # wich is the under_test_prs after parsing
+        under_test_prs = [self.__pr_list[0]]
+        # prs haven been parsed, for preventing infinite loop
+        parsed_prs = []
+        while tmp_under_test_prs:
+            parsed_prs.extend(tmp_under_test_prs)
+            # the next tmp_under_test_prs
+            tmp_tmp_under_test_prs = []
```

Comment:
what these variables 'tmp_tmp_under_test_prs' 'tmp_under_test_prs' under_test_prs ''  used for,  could their intention be reflected in the naming?